Philippines/bago/bago.md

```diff
@@ -0,0 +1,3 @@
+Dito nag-uumpisa ang iyong daan tungo sa pagbabago.
+
+Bumalik sa Udacity para mag-aral o kaya bumalik sa [umpisa] (Philippines/salamat.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.